python/cvmfs/_common.py

```diff
@@ -75,17 +75,17 @@ def _close(self):
 
 
 
-class DatabaseObject(CompressedObject):
+class DatabaseObject:
     db_handle_ = None
 
-    def __init__(self, compressed_db_file):
-        CompressedObject.__init__(self, compressed_db_file)
+    def __init__(self, db_file):
+        self.file_ = db_file
```

Comment:
General remark for consistency: We should agree on how to mark private members. I think I used a trailing underscore everywhere until now. In your `Cache` class I've seen a prepended underscore... Let's not mix that.